BSP/STM32F413/Makefile

```diff
@@ -45,7 +45,8 @@ $(wildcard $(RTOS_DIR)/uC-Serial/Driver/ST/*.c)	\
 $(wildcard $(RTOS_DIR)/uC-Serial/Line/*.c)	\
 $(wildcard $(RTOS_DIR)/uC-Serial/OS/*.c)	\
 $(wildcard $(RTOS_DIR)/uC-Serial/OS/uCOS-III/*.c)	\
-$(wildcard $(RTOS_DIR)/uC-Serial/Source/*.c)	
+$(wildcard $(RTOS_DIR)/uC-Serial/Source/*.c)	\
+$(wildcard $(TOP_DIR)/Config/Src/*.c) 
```

Comment:
Maybe add a comment that you are taking advantage of this being last in C_SOURCES (and how) in case we ever edit this in the future